soh/soh/SohGui/SohMenuEnhancements.cpp

```diff
@@ -673,10 +674,11 @@ void SohMenu::AddMenuEnhancements() {
                               " - Not within range of Ocarina Playing spots."));
 
     AddWidget(path, "Masks", WIDGET_SEPARATOR_TEXT);
-    AddWidget(path, "Bunny Hood Effect", WIDGET_CVAR_CHECKBOX)
+    AddWidget(path, "Bunny Hood Effect", WIDGET_CVAR_COMBOBOX)
         .CVar(CVAR_ENHANCEMENT("MMBunnyHood"))
-        .Options(CheckboxOptions().Tooltip("Wearing the Bunny Hood grants a speed increase link in Majora's Mask. "
-                                           "The longer jump option is not accounted for in Randomizer logic.\n\n"
+        .Options(ComboboxOptions().ComboMap(bunnyHoodEffectMap).Tooltip("Wearing the Bunny Hood grants a speed increase like in Majora's Mask.\n"
+                                           "Can also include a longer jump.\n"
+                                           "The effects of either option are not accounted for in Randomizer logic.\n"
```

Comment:
The tooltip isn't entirely accurate. MM has the bunny hood speed + jump not just the increased run speed, which is why it was originally introduced with both the run speed and jump. And then later the jump speed was made optional because it was just too strong in OoT.